legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/db2Extension.pure

```diff
@@ -31,7 +32,11 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::db2::
       selectSQLQueryProcessor = processSelectSQLQueryForDB2_SelectSQLQuery_1__SqlGenerationContext_1__Boolean_1__String_1_,
       columnNameToIdentifier = columnNameToIdentifierDefault_String_1__DbConfig_1__String_1_,
       identifierProcessor = processIdentifierWithDoubleQuotes_String_1__DbConfig_1__String_1_,
-      dynaFuncDispatch = $dynaFuncDispatch
+      dynaFuncDispatch = $dynaFuncDispatch,
+      createTableSQLQueryProcessor = processCreateTempTable_CreateTableSQL_1__DbConfig_1__String_MANY_,
```

Comment:
why is createTable pointing to create temp table?

Reply:
Because CreateTableSQLQueryProcessor is a function that takes in a CreateTableSQL which has the a property isTempTable which we set to true. that is why we are pointing CreateTable to create temp table

Reply:
In that case, the fn should be named createTable, and it should contain logic for both creating normal table, as well as temp table conditioned on isTempTable.